srcpkgs/oneko/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'oneko'
+pkgname=oneko
+version=1.2.sakura.5
+revision=1
+hostmakedepends="pkg-config"
+makedepends="libX11-devel libXext-devel"
+short_desc="Cat that chases around your cursor"
+maintainer="wael <40663@protonmail.com>"
+license="Public Domain"
+homepage="http://www.daidouji.com/oneko"
+distfiles="http://www.daidouji.com/oneko/distfiles/oneko-${version}.tar.gz"
+checksum=2c2e05f1241e9b76f54475b5577cd4fb6670de058218d04a741a04ebd4a2b22f
+
+do_build() {
+	${CC} ${CFLAGS} -DSHAPE oneko.c -o oneko $(pkg-config --libs xext x11) -lm
+}
+
+do_install() {
+	vbin oneko
+	vman oneko.man oneko.1
+	vman oneko.man.jp oneko.jp.1
+	vdoc README
+	vdoc README-NEW
+	vdoc README-SUPP
```

Comment:
They are in japanese and in ISO-2022-JP and I don't think they are useful, so we shouldn't install them.

Reply:
could be useful for the 1 japanese person that decides to read it. but alright.

Reply:
does it provide info about how to use oneko that isn't in the manual pages? if not, i don't think it's very relevant to the package

Reply:
i think `oneko --help` and the man pages is enough.